wordpress-mobile/gutenberg-mobile
# Updated GB hash to fix rotation problems on Android

Fix is in the Gutenberg PR
https://github.com/WordPress/gutenberg/pull/14563

_WILL ADD WPAndroid PR Link_

There was a red screen when rotating the screen on Android with an empty paragraph.

To test:
1. start a draft
2. do not enter any text
3. rotate the screen
4. observe it doesn't crash anymore
